TrackletAlgorithm/MatchEngineUnit_parameters.h

```diff
@@ -28,9 +28,28 @@ static bool nearFullUnitBool(ap_uint<kNBitsBuffer> rptr, ap_uint<kNBitsBuffer> w
   return wptr1==rptr || wptr2==rptr;
 }
 
+template<int kNBitsBuffer>
+class nearFull3Class {
```

Comment:
CMS coding rules state that class names should start with a capital letter, whereas variable and function names should start with a lower case letter.